src/studio-home/tabs-section/libraries-tab/index.tsx

```diff
@@ -146,21 +146,22 @@ interface LibrariesListProps {
   handleCheck?: (library: LibraryV1Data, action: 'add' | 'remove') => void;
   setSelectedLibraries?: (libraries: LibraryV1Data[]) => void;
   hideMigationAlert?: boolean;
-  initialFilter?: Filter[];
+  migrationFilter: Filter[];
+  setMigrationFilter: React.Dispatch<React.SetStateAction<Filter[]>>;
```

Comment:
Could you maybe add a comment here to explain why this state is a prop? (to preserve the state across different steps of the flow) Because it doesn't seem like it should be a prop.

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/2550/commits/0ed45398e993e5bb9dad2c8c01fd28cbbf182727